lightwood/analysis/explain.py

```diff
@@ -94,18 +93,20 @@ def explain(data: pd.DataFrame,
 
             # base ICP
             X = deepcopy(icp_X)
+            # Ugly optimization but accessing `values` actually wasted a lot of time
```

Comment:
Remove this comment. `.values` is a standard technique and we don't need a post mortem

Reply:
yes but assigning `.values` to something isn't. The weird behavior here is that `.value` actually seems to => some function invocation in the background, it's not just a reference